lib/RateLimiterRedis.js

```diff
@@ -56,6 +75,10 @@ class RateLimiterRedis extends RateLimiterStoreAbstract {
 
   _upsert(rlKey, points, msDuration, forceExpire = false) {
     return new Promise((resolve, reject) => {
+      if (!this._isRedisReady()) {
+        return reject(new Error('Not connected'));
```

Comment:
Should it be more self-descriptive? Something like `Redis connection is not ready`. And the same for below error message.